Samples/DotNetCore/XenaSamples.HybridFlow.MvcClient/Xena/XenaHttpClient.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace XenaSamples.HybridFlow.MvcClient.Xena
+{
+    public class XenaHttpClient
+    {
+        private readonly string _xenaEndpoint;
+        private static readonly HttpClient client = new HttpClient();
+
+        public XenaHttpClient(string xenaEndpoint)
+        {
+            _xenaEndpoint = xenaEndpoint;
+        }
+
+        public async Task<string> GetAsyc(string accessToken, string apiEndPoint)
+        {
+           client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", accessToken);
+           return await client.GetStringAsync($"{_xenaEndpoint}{apiEndPoint}");
+        }
+
+        public static XenaHttpClient Init(XenaApiOptions options)
+        {
+            var response = new XenaHttpClient(options.XenaEndpoint);
+            return response;
```

Comment:
To små detaljer: response => client og Init => Create.
Hvorfor har du både en factory method og en public constructor?

Reply:
I thought i would add more options but i dont want to over complicate things so removed it.